src/pages/en/core-concepts/astro-components.md

```diff
@@ -41,18 +41,22 @@ import Button from './Button.astro';
 
 ### The Component Script
 
-Astro uses a code fence (`---`) to identify the component script in your Astro component. If you've ever written Markdown before, you may already be familiar with a similar concept called *front matter.* Astro's idea of a component script was directly inspired by this concept.
+Astro uses a code fence (`---`) to identify the component script in your Astro component. If you've ever written Markdown before, you may already be familiar with a similar concept called *frontmatter.* Astro's idea of a component script was directly inspired by this concept.
 
 You can use the component script to write any JavaScript code that you need to render your template. This can include:
 
-- Importing other Astro components
+- Importing other Astro components 
 - Importing other framework components, like React
 - Importing data, like a JSON file
 - fetching content from an API or database
 - creating variables that you will reference in your template
 
+
+>⚠️ In Astro v0.21, a bug was introduced that requires imports inside components to be at the top of your component script.
+
 ```astro
 ---
+// Import components at the top of your script
```

Comment:
```suggestion
```astro
---
// Note: Imports must live at the top of your file.
```

Reply:
I generally try to avoid calling things bugs in our docs, if they can instead be framed as "requirements". 

The migration guide is a bit different since it's literally recording changed behavior between specific versions, which is why it probably used the term "new bug" more liberally.